web/org.eclipse.xtext.web.example.statemachine/src/org/eclipse/xtext/web/example/statemachine/GenerateStatemachine.mwe2

```diff
@@ -84,6 +84,8 @@ Workflow {
 				fragment = parser.antlr.XtextAntlrUiGeneratorFragment auto-inject {}
 			}
 			
+			fragment = parser.antlr.XtextAntlrGeneratorComparisonFragment auto-inject {}
```

Comment:
org.eclipse.xtext.tests/src/org/eclipse/xtext/TestLanguagesFragments.mwe2 would be a good start